docker-compose.yml

```diff
@@ -7,10 +7,35 @@ services:
     restart: always
     ports:
       - "8888:3000"
+    labels:
+      - "traefik.enable=true"
+      - "traefik.basic.frontend.rule=Host:explorer.raiden.network"
```

Comment:
This should use the `${SERVER_NAME}` variable (or remove it from .env if it doesn't need to be configurable)